Added more Python Programs with implementation

rutujak24 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added more Python Programs with the implementation that is easy to understand and all the problems statements are supported by easy python solutions for it.

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

rutujak24 committed:
Added more Python Programs with implementation

siddharth25pandey commented:
@rutujak24 Please create Separate PR for different issues.
Also, mention the Issue number in the PR Template

rutujak24 commented:
@siddharth25pandey  I have added 4-5 python implemented codes to the "other" folder.
There isn't any issue listed out there. I just wanted to make the python folder vast with codes for beginner to advanced, I have added those.

siddharth25pandey commented:
@rutujak24 You can create a PR on specific issue only.
Without any mentioned issue, we can't merge your PR.
Hope you Understand.
Therefore I am closing this PR
Thanks!
